app/models/pipeline_run.rb

```diff
@@ -431,7 +431,13 @@ def taxon_byte_ranges_available?
   end
 
   def align_viz_available?
-    # TODO(tiago): we should not have to access aws
+    if Rails.env == "test"
+      # Otherwise, tests will raise a WebMock::NetConnectNotAllowedError.
+      # TODO: (gdingle): see IDSEQ-2602
+      return true
+    end
+
+    # TODO(tiago): we should not have to access aws. see IDSEQ-2602.
```

Comment:
Good catch! Thank you for opening the ticket. Definitely, we need to remove this... I wonder how much time is adding... but should be non-negligible.

Reply:
I don't know much about this feature or else I would take a stab at fixing it here.

Reply:
Yeah, I would have to see where it is used, too. I think the ticket makes sense and we schedule it on the next sprint.
One question on the ticket: would you call this a bug?

Reply:
I would call this tech debt but not a bug because it does not have a known user impact. Tech debt I think we are categorizing as stories.